### PR TITLE
Mapped render function for flashing TNT block

### PR DESCRIPTION
**Pull request by shartte**
Mapped render function for flashing TNT block

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping [hunk 1]
```diff
@@ -1 +1,10 @@
 CLASS net/minecraft/class_957 net/minecraft/client/render/entity/TntMinecartEntityRenderer
+	METHOD method_23190 renderFlashingBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IZ)V
+		COMMENT Renders a given blockstate into the given buffers either normally or with a bright white overlay.
```
Comment (Juuxel) [inline mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping]:
```suggestion
		COMMENT Renders a given block state into the given buffers either normally or with a bright white overlay.
```

File: mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping [hunk 2]
```diff
@@ -1 +1,10 @@
...
+		COMMENT Used for rendering primed TNT either standalone or as part of a TNT minecart.
+		COMMENT @param drawFlash If true, a white semi-transparent overlay is added to the block to indicate the flash.
```
Comment (Juuxel) [inline mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping]: This should be a javadoc comment on the parameter itself (select the parameter in Enigma and add docs to that).

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping (hunk 2 above)
Comment (shartte) [reply mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping]: Learning something new everyday :D

**Comment by shartte** [comment]
"Block state" over "blockstate" reads .. weird, but if thats the convention that's fine. I corrected the PR.

File: mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping [hunk 3]
```diff
@@ -1 +1,10 @@
 CLASS net/minecraft/class_957 net/minecraft/client/render/entity/TntMinecartEntityRenderer
+	METHOD method_23190 renderFlashingBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IZ)V
+		COMMENT Renders a given block state into the given buffers either normally or with a bright white overlay.
+		COMMENT Used for rendering primed TNT either standalone or as part of a TNT minecart.
+		ARG 0 blockState
+		ARG 1 matrices
+		ARG 2 vertexConsumers
+		ARG 3 light
+		ARG 4 drawFlash
+			COMMENT If true, a white semi-transparent overlay is added to the block to indicate the flash.
```
Comment (liach) [inline mappings/net/minecraft/client/render/entity/TntMinecartEntityRenderer.mapping]:
```suggestion
			COMMENT whether a white semi-transparent overlay is added to the block to indicate the flash
```
Note that in fabric yarn we use phrases than full sentences for javadoc tag comments like for `@param` `@return` etc

**Comment by liach** [comment]
> 
> 
> "Block state" over "blockstate" reads .. weird, but if thats the convention that's fine. I corrected the PR.

In code we use `blockState` than `blockstate` because they are two words. They are separated in mojang's update log or minecraft wiki as well.

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**